coral-schema/src/test/resources/base-date-type-field.avsc

```diff
@@ -0,0 +1,12 @@
+{
+  "type" : "record",
+  "name" : "basedatetypefield",
+  "namespace" : "coral.schema.avro.base.date.type.field",
+  "fields" : [ {
+    "name" : "Date_Field",
+    "type" : [ "null", {
+      "type" : "int",
+      "logicalType" : "date"
+    } ]
```

Comment:
is this the hive schema representation of 'Date' data type column? who injects the 'logicalType'? why is this not just 
```
{
          "name" : "Date_Field",
          "type" : [ "null", "date" ]
        },
```

Reply:
This is Avro schema representing `date` type, we don't have `"type" : [ "null", "date" ]` in Avro.